playbooks/roles/load-balancer-v2/tasks/site-is-down.yml

```diff
@@ -0,0 +1,37 @@
+---
+# This set of tasks is meant to be run like so:
```

Comment:
This is a nice playbook! :)
Can you update [the docs on this](https://gitlab.com/opencraft/documentation/private/blob/master/howtos/ocim/creating_maintenance_pages.md)?

Reply:
Sure! I created https://gitlab.com/opencraft/documentation/private/merge_requests/422 .

As far as I know, those instructions are up-to-date, so I just made a note that you have to make the changes in the repository and linked to the deploy instructions on https://gitlab.com/opencraft/documentation/private/blob/master/ops/incidents.md#we-need-to-put-up-a-maintenance-page-for-some-site